### PR TITLE
Fix bug with deleting unshared stories after reload storyteller

### PR DESCRIPTION
**Pull request by val-fed**

### CODE REVIEW

File: tools/storyteller/StoryWriter.java [hunk 1]
```diff
@@ -41,11 +41,14 @@ public StoryWriter(StorytellerConfig storytellerConfig, FileUtils fileUtils) {
     allStories = new ArrayList<>();
   }
 
-  void startStory(String project) {
+  void startStory(String project, List<Story> stories) {
     currentStoryBuilder = Story.newBuilder();
     currentStoryBuilder
         .setStartTimeMs(getCurrentTimestamp())
         .setProject(project);
+    if(allStories.isEmpty()) {
```
Comment (oferb) [inline tools/storyteller/StoryWriter.java]: Please add a space after `if`

File: tools/storyteller/StoryWriter.java [hunk 2]
```diff
@@ -41,11 +41,14 @@ public StoryWriter(StorytellerConfig storytellerConfig, FileUtils fileUtils) {
     allStories = new ArrayList<>();
   }
 
-  void startStory(String project) {
+  void startStory(String project, List<Story> stories) {
```
Comment (oferb) [inline tools/storyteller/StoryWriter.java]: Maybe add a separate method `addStories()` instead?

File: tools/storyteller/StoryWriter.java [hunk 3]
```diff
@@ -50,16 +50,22 @@ void startStory(String project) {
     saveStories();
   }
 
+  void addStories(List<Story> stories){
+    if (allStories.isEmpty()) {
```
Comment (oferb) [inline tools/storyteller/StoryWriter.java]: Why only if it's empty?

Comment (val-fed) [reply tools/storyteller/StoryWriter.java]: `allStories` variable was not reset during the work of the Storyteller. And when we pressed the `pause` button `allStories` did not reset.
I changed behavior in the next commit. Now, regardless of whether Storyteller was closed completely or the pause button was pressed `allStories` is reset.

File: tools/storyteller/Storyteller.java [hunk 4]
```diff
@@ -197,6 +197,7 @@ YearMonth getLastMonth(Instant instant) {
   }
 
   public void startup(String project) {
+    writer.addStories(getUnsharedStories());
```
Comment (oferb) [inline tools/storyteller/Storyteller.java]: Maybe you should put this in the constructor, so it's called only once. Would that fix the issue? Then you don't need to `allStories = new ArrayList<>();` in `addStories()`.

**Pull request merged**